sklearn/decomposition/factor_analysis.py

```diff
@@ -347,3 +359,36 @@ def score(self, X, y=None):
             Average log-likelihood of the samples under the current model
         """
         return np.mean(self.score_samples(X))
+
+    def _rotate(self, method="varimax", n_components=None, tol=1e-6):
+        "Rotate the factor analysis solution."
+        implemented = {"varimax", "quartimax"}
+        if method in implemented:
+            self.components_ = _ortho_rotation(
+                self.components_.T, method=method, eps=tol)[:n_components]
+        else:
+            raise ValueError("'method' must be in %s, not %s"
+                             % (implemented, method))
```

Comment:
implemented is a set, so its printing order is not deterministic. This doesn't seem a great idea to me.